app/controllers/state_file/questions/eligible_controller.rb

```diff
@@ -5,6 +5,7 @@ class EligibleController < QuestionsController
 
       def edit
         super
+        @vita_link = vita_link
```

Comment:
@rickreyhsig good point about using shared concerns! I forgot about the `OtherOptionsLinksConcern` that has those airtable links

Reply:
Perfect!